pkgs/development/python-modules/consul2/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi
+, requests, six, pytest }:
+
+buildPythonPackage rec {
+  pname = "python-consul2";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-KchZ3nPhfzarmb6DH9wdkk8W6HciMyd6KO+St7mZlc0=";
+  };
+
+  buildInputs = [ requests six pytest ];
```

Comment:
These need to be `propagatedBuildInputs` for python packages, else they will not be included as runtime dependencies.

Also, I doubt `pytest` is a dependency for anything other than the `checkPhase`, which we're you're not currently running.

Reply:
Good point about `buildInputs`. As for tests, you're right, I should be using `checkInputs` for that. And I will see if I can actually make the tests work.

Reply:
Removed `pytest`.